mpv.py

```diff
@@ -512,6 +518,29 @@ def playlist_next(self, mode='weak'):
     def playlist_prev(self, mode='weak'):
         self.command('playlist_prev', mode)
 
+    @property
+    def playlist_position(self):
+        val = self._get_property('playlist-pos')
+        return int(val) if val else None
+
+    @property
+    def playlist_length(self):
+        return self._get_property('playlist-count', proptype=int)
+
+    @playlist_position.setter
+    def playlist_position(self, new):
+        if new >= self.playlist_length or new < 0:
+            raise ValueError("Trying to access out of bounds index %d in mpv playlist" % new)
+        else:
+            self._set_property("playlist-pos", new, proptype=int)
+
+    @property
+    def filenames():
```

Comment:
This looks useful. Only I don't like the name, as it doesn't say that this is about the playlist (instead of just the currently playing file as everything else).